control/tap-ctl-unpause.c

```diff
@@ -54,8 +55,8 @@ tap_ctl_unpause(const int id, const int minor, const char *params, int flags,
 	message.u.params.flags = flags;
 
 	if (params)
-		strncpy(message.u.params.path, params,
-				sizeof(message.u.params.path) - 1);
+		safe_strncpy(message.u.params.path, params,
+			     sizeof(message.u.params.path) - 1);
```

Comment:
The previous code was safe as it left one byte at the end for the nul terminator and previously set the entire struct to 0.

In any case, this should now be `sizeof(message.u.params.path)`, not `sizeof(message.u.params.path) - 1`.

Reply:
That's what I get for searching and replacing :(